base/server/examples/installation/ca-clone-w-p12.cfg

```diff
@@ -0,0 +1,35 @@
+[DEFAULT]
+pki_server_database_password=Secret.123
+pki_cert_chain_path=ca_signing.crt
+
+[CA]
+pki_admin_email=caadmin@example.com
+pki_admin_name=caadmin
+pki_admin_nickname=caadmin
+pki_admin_password=Secret.123
+pki_admin_uid=caadmin
+
+pki_client_pkcs12_password=Secret.123
+
+pki_ds_base_dn=dc=ca,dc=pki,dc=example,dc=com
+pki_ds_database=ca
+pki_ds_password=Secret.123
+
+pki_security_domain_hostname=primary.example.com
+pki_security_domain_https_port=8443
+pki_security_domain_user=caadmin
+pki_security_domain_password=Secret.123
+
+pki_ca_signing_nickname=ca_signing
+pki_ocsp_signing_nickname=ca_ocsp_signing
+pki_audit_signing_nickname=ca_audit_signing
+pki_sslserver_nickname=sslserver
+pki_subsystem_nickname=subsystem
+
+pki_clone=True
+pki_clone_replicate_schema=True
+pki_clone_uri=https://primary.example.com:8443
+
+# To clone with PKCS #12 file, specify the following parameters:
```

Comment:
This comment is probably not needed for this file.

Reply:
@edewata How about I change the comment to something like "The following parameters are to enable cloning by using PKCS # 12 "?